frontend/test/metabase/scenarios/binning/qb-implicit-joins.cy.spec.js

```diff
@@ -0,0 +1,86 @@
+import { restore, popover } from "__support__/e2e/cypress";
+
+describe("scenarios > binning > from a saved QB question using implicit joins", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.intercept("POST", "/api/dataset").as("dataset");
+  });
+
+  context("via simple question", () => {
+    beforeEach(() => {
+      cy.visit("/question/1");
+      cy.findByText("Summarize").click();
+    });
+
+    it("should work for time series", () => {
+      cy.findByTestId("sidebar-right").within(() => {
+        openPopoverFromDefaultBucketSize("Birth Date", "by month");
+      });
+
+      popover().within(() => {
+        cy.findByText("Year").click();
+      });
+
+      waitAndAssertOnRequest("@dataset");
+
+      cy.findByText("Count by User → Birth Date: Year");
+
+      // The default chosen "visualization" is table
+      cy.findByText("1958");
+      cy.findByText("313");
+    });
+
+    it("should work for number", () => {
+      cy.findByTestId("sidebar-right").within(() => {
+        openPopoverFromDefaultBucketSize("Price", "Auto bin");
+      });
+
+      popover().within(() => {
+        cy.findByText("10 bins").click();
+      });
+
+      waitAndAssertOnRequest("@dataset");
+
+      cy.findByText("Count by Product → Price: 10 bins");
+      cy.findByText("196");
+    });
+
+    it.skip("should work for longitude", () => {
+      cy.findByTestId("sidebar-right").within(() => {
+        openPopoverFromDefaultBucketSize("Longitude", "Auto bin");
+      });
+
+      popover().within(() => {
+        // Test fails at this point (UI inconsistency).
+        // It simply gives `10°` as the option, which is different than in any other longitude binning scenario.
+        cy.findByText("Bin every 10 degrees").click();
```

Comment:
Can we test something different than the [auto-default](https://www.metabase.com/docs/latest/operations-guide/environment-variables.html#mb_breakout_bin_width) - like `20°` instead?

Reply:
I can change this, but don't see the difference, tbh. We're still **explicitly** chooseing the bucket size. Np. Let's go with the 20.

Reply:
Done in the latest version of the test.